src/components/Collections/CollectionsIndexHeader.js

```diff
@@ -11,38 +11,63 @@ import { connect } from 'react-redux';
 
 import Colors from '../../constants/Colors';
 import CollectionsDialog, { COLLECTIONS_DIALOG_ACTIONS, CollectionsDialogText } from '../Dialog/CollectionsDialog';
-import { collectionsCounterSelector } from '../../redux/selectors';
+import { collectionsCounterSelector,creatingCollectionSelector  } from '../../redux/selectors';
 import HeaderCountIcon from '../Icons/HeaderCountIcon';
+import { isAddingNewCollection } from '../../redux/action-creators';
 
 const AddCollectionButton = ({ onPress }) => (
   <TouchableOpacity onPress={onPress}>
     <Feather name="plus-square" size={24} color={Colors.headerIcon} />
   </TouchableOpacity>
+
+);
+const AddCollectionInputButton = ({ onPress }) => (
+  <TouchableOpacity onPress={onPress}>
+    <Feather name="plus-square" size={24} color={Colors.headerIcon} />
+  </TouchableOpacity>
+
+);
+
+const SearchInputButton = ({ onPress }) => (
+  <TouchableOpacity onPress={onPress}>
+    <Feather name="search" size={24} color={Colors.headerIcon} />
+  </TouchableOpacity>
+
 );
 
 AddCollectionButton.propTypes = {
   onPress: func.isRequired,
 };
 
-const CollectionsIndexHeader = ({ showNewCollectionButton, collectionsCounter }) => {
+const CollectionsIndexHeader = ({ showNewCollectionButton, collectionsCounter, navigation, isAddingNewCollectionAction, creatingCollection }) => {
+
   const [collectionsDialogText, setCollectionsDialogText] = useState(null);
   const totalCollectionsCount = collectionsCounter.customCount + collectionsCounter.preBuiltCount;
 
   const handleNewCollectionPress = () => {
     setCollectionsDialogText(CollectionsDialogText[COLLECTIONS_DIALOG_ACTIONS.CREATE]);
   };
+  const toNewCollectionInput = () => {
```

Comment:
It's often good practice to use `handle{xyz}Press` ahead of press or click actions. It'll make them easier to distinguish from other functions. Since `handleNewCollectionPress` is no longer used, that would probably be a good name.

Reply:
renamed